values.yaml

```diff
@@ -489,6 +493,9 @@ etcd-operator:
   customResources:
     # create default etcd cluster
     createEtcdClusterCRD: true
+  image:
+    # Image pull policy. Change to "IfNotPresent" when switching to stable images
+    pullPolicy: IfNotPresent
```

Comment:
```suggestion
```
The registry is quay.io for the etcd and `pullPolicy: IfNotPresent` in the upstream chart, no reason to include it here.